### PR TITLE
Log inprogress calls

### LINKED ISSUE #60: Allow multiple users to use dashboard at the same time
When multiple users are using the dashboard at the same time, they might be calling the same patient. Might be better to change the dashboard so that it contains a status 'in_progress' for each patient, where the caller sets the patient to in_progress before starting to call them.

Maybe best approach is:
- on Patient select screen phone number is blocked
- When pressing on 'start' phone number is shown and status is in progress
- When filling in result of call, progress is reset

### PR DESCRIPTION
**Pull request by WoltersEric**
closes #60 
Fixes bug caused by current_patient_nmbr.call_number being initialized as None
Fixed bug when switching to new date didn't reset pred_idx

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.0.8] - 2024-03-21
+### Changed
+- Dashboard now logs when a call has started to prevent duplicate calls
+
```
Comment (rubenpeters91) [inline CHANGELOG.md:12]: plus bugfix when going to different day right?

File: src/noshow/dashboard/helper.py [hunk 2]
```diff
@@ -7,6 +7,55 @@
 from noshow.database.models import ApiCallResponse, ApiPatient
 
 
+def render_patient_info(
+    Session: sessionmaker,
```
Comment (rubenpeters91) [inline src/noshow/dashboard/helper.py:11]: Is sessionmaker a type?

**Review by rubenpeters91 — APPROVED** [review]

File: src/noshow/dashboard/helper.py (hunk 2 above)
Comment (WoltersEric) [reply src/noshow/dashboard/helper.py:11]: yes

Comment (WoltersEric) [reply src/noshow/dashboard/helper.py:11]: yes

File: CHANGELOG.md (hunk 1 above)
Comment (WoltersEric) [reply CHANGELOG.md:12]: added it

**Pull request merged**